### PR TITLE
docs: mention something about debugging PRs

### PR DESCRIPTION
**Pull request by matz-e**

### CODE REVIEW

File: deploy/docs/debug_bb5.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+   number:
+   ```console
+   export PR_NUMBER=1234
+   export PR_STAGE=applications
+   export PR_BASE=$(readlink -f /gpfs/bbp.cscs.ch/ssd/apps/hpc/jenkins/pulls/${PR_NUMBER}/deploy/${PR_STAGE}/latest)
+   ```
+2. Move your own configuration files out of the way. If the official
+   instructions are following, one can use:
+   ```console
+   for fn in ${SPACK_ROOT}/etc/spack/*.yaml; do mv ${fn}{,.old} done
```
Comment (GianlucaFicarelli) [inline deploy/docs/debug_bb5.md]: Missing `;`
```suggestion
   for fn in ${SPACK_ROOT}/etc/spack/*.yaml; do mv ${fn}{,.old}; done
```

File: deploy/docs/debug_bb5.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+   ```
+3. Copy the environment that the CI is attempting to build somewhere under
+   your control:
+   ```console
+   cp -R ${PR_BASE}/data/build_enironment .
```
Comment (GianlucaFicarelli) [inline deploy/docs/debug_bb5.md]: Typo
```suggestion
   cp -R ${PR_BASE}/data/build_environment .
```

**Review by pramodk — DISMISSED** [review]
thats very useful! thanks @matz-e !

**Review by GianlucaFicarelli — COMMENTED** [review]
I'll probably try the commands soon, thanks for adding these info.

**Comment by matz-e** [comment]
> I'll probably try the commands soon, thanks for adding these info.

Thanks for the fixes! Happens when I don't strictly copy and paste from my history…

**Review by GianlucaFicarelli — APPROVED** [review]

**Pull request merged**